.github/workflows/codedeploy-staging-cd.yml

```
@@ -0,0 +1,47 @@
+name: AWS Codedeploy CD
+
+# Controls when the action will run.
+on:
+  push:
+    branches: [master]
```

```suggestion
    branches: [master]
    tags: ['test/**']
```

Maybe we could include an alternative for master branch, like tags (or alternative branch) pattern. The suggestion would match any tag starting with `test/`.
This would allow pushing a tag for testing some unfinished or otherwise experimental branches/commits, that are not ready for merging to master.

This would obviously require coordination between the team, so we don't all try to push tags to different places and pushing to master at the same time. Using the tags would be the special case: when using a tag, let others know so they don't push new code to master.

---

Now that I think about it I think I'll rather make the sandbox environment ONLY deploy on pushes to a staging-sandbox branch as it communicates that there is only one branch and one environment and eliminates some need for developer communication. This also stops any merge into master from interfering with someones testing in a `test/**` commit due to the master deployment potentially overwriting the branch being tested

---

This is fine also, but it still requires coordination in the same situations. The coordination would just be: who will push to the staging branch.

The main benefit would be, that it's easier to see what is running in the staging.